libs/contract/src/lib/bucket/+state/bucket.service.ts

```diff
@@ -120,7 +120,7 @@ export class BucketService extends CollectionService<BucketState> {
       await this.contractService.addNegotiation(contractId, {
         ...contract,
         ...commonFields
-      })
+      }, false)
```

Comment:
remove false as you don't use it : 
```suggestion
      })
```